Bottom: b4eb52c1b720fbd6e30ad1b2634ad6dc61284f45
Top:    cccc9417435ab274dd4973f1a3df3b15b66cdc6a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 18:54:45 +0000

Allow running of action scripts when running PCB as an exporter


---

diff --git a/src/main.c b/src/main.c
index 162a0e0..e3c164f 100644
--- a/src/main.c
+++ b/src/main.c
@@ -996,12 +996,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  if (gui->printer || gui->exporter)
-    {
-      gui->do_export (0);
-      exit (0);
-    }
-
   /*    FIX_ME
      LoadBackgroundImage (Settings.BackgroundImage); */
 
@@ -1034,6 +1028,12 @@ main (int argc, char *argv[])
       hid_parse_actions (Settings.ActionString, 0);
     }
 
+  if (gui->printer || gui->exporter)
+    {
+      gui->do_export (0);
+      exit (0);
+    }
+
 #if HAVE_DBUS
   pcb_dbus_setup();
 #endif
